### ISSUE TITLE
This vs. Sphinx AutoAPI

### ISSUE THREAD

**Issue by ATidmore** [carlos-jenkins/autoapi#8]
What's the difference between this code base and https://sphinx-autoapi.readthedocs.io ?? It's confusing to know which to implement for auto-documenting functionality.

**Comment by ben-spiller** [comment]
Agreed the naming of these projects really confused me too - https://github.com/carlos-jenkins/autoapi and https://github.com/readthedocs/sphinx-autoapi seem to be completely unrelated projects, though with a very similar purpose and identical names. Renaming this project or adding a note on the front page to avoid confusion between them would be really helpful.

**Comment by mjstahlberg** [comment]
I'm also regularly confused about this and end up installing `sphinx-autoapi` instead of this one (`autoapi`). It doesn't help at all that `sphinx-autoapi`'s extension is called `autoapi.extension` while `autoapi`'s is called `autoapi.sphinx`…

Since `sphinx-autoapi` is larger and vastly more popular at this point, I would suggest picking a nice new name for this package so it doesn't get buried in search results or overlooked in recommendations.

There are good reasons to choose this package over the other; in particular this package lets you procedurally generate some of your docstrings (alongside procedurally generated classes, say) as the code is imported while `sphinx-autoapi` does not import or run code.
